src/Illuminate/Mail/Transport/MailgunTransport.php

```diff
@@ -187,4 +187,25 @@ public function setDomain($domain)
     {
         return $this->domain = $domain;
     }
+
+    /**
+     * Get the API end-point being used by the transport.
+     *
+     * @return string
+     */
+    public function getEndpoint()
+    {
+        return $this->endpoint;
+    }
+
+    /**
+     * Set the API end-point being used by the transport.
+     *
+     * @param  string  $endpoint
+     * @return string
+     */
+    public function setEndpoint($endpoint)
```

Comment:
I think a setter method should return void or $this

Reply:
Yeah, but all the other setter methods (key and domain) return the property value, so I left it the same for the endpoint setter.